### ISSUE TITLE
academic grading, small improvement

### ISSUE THREAD

**Issue by johannes-titz** [shevandrin/qti#141]
I think there should be two labels. The message at the top is usually not identical to the grade. E.g. my current message is: "Vorläufige Note nach automatischer Auswertung:", whereas for the table, I just would like "Note". One option would be to add another label for the table. It can be identical as default.

Also, I believe it makes more sense to have grade 1,0 at the top and 5,0 at the bottom. What is your opinion on that?

**Comment by shevandrin** [comment]
Good idea. I would suggest to derive label with two items: table_label and grade_label. Should we have one slot for both or keep them independent in two separate slots?

**Comment by johannes-titz** [comment]
I would prefer two slots.

**Comment by shevandrin** [comment]
![image](https://github.com/shevandrin/qti/assets/43883261/c63f3df1-30f0-4ec8-9c8f-ee7c33148d36)

it is done

**Comment by johannes-titz** [comment]
Great! One short question: this is normal html so in principle we could style the table, right? Do you use a specific function to generate the html?

**Comment by shevandrin** [comment]
> Great! One short question: this is normal html so in principle we could style the table, right? Do you use a specific function to generate the html?

i construct table as a set of tags via htmltools package, as all other xml tags as well

**Comment by johannes-titz** [comment]
I would suggest` kableExtra::kable_styling(knitr::kable(our_table), position = "left", full_width = F)`, which seems to work out of the box. I tried on a feedback object. Maybe needs some tweaking. Only knitr:kable might look to crude, but I have not seen it yet.

**Comment by shevandrin** [comment]
![image](https://github.com/shevandrin/qti/assets/43883261/32411064-f14a-4fb2-bcd2-ea873342be8e)

i like how it looks like
